user/utils.py

```diff
@@ -0,0 +1,34 @@
+import json
+import jwt
+import my_settings
+
+from django.http import JsonResponse
+
+from .models import User
+from my_settings  import SECRET,ALGORITHM
+
+def token_check(func): 
+    def wrapper(self, request, *args, **kwargs):
+
+        if "AUTHORIZATION" not in request.headers: 
+            return JsonResponse({"message" : "INVALID_LOGIN"}, status=400)
+
+        encode_token = request.headers["AUTHORIZATION"] 
+
+        try:
+            user_id     = jwt.decode(encode_token, SECRET, algorithm = ALGORITHM)                         
```

Comment:
jwt.decode() 함수가 실행되면 access_token의 payload 부분의 값이 복호화되어 리턴됩니다.
그렇게 때문에 user_id라는 변수에 지정하는 것 보다는 payload에 할당해주는 것이 명확합니다.

```suggestion
            payload = jwt.decode(encode_token, SECRET, algorithm = ALGORITHM)                         
```

Reply:
**필수**